modules/web/js/ballerina/components/package-definition.jsx

```diff
@@ -142,10 +142,18 @@ class PackageDefinition extends React.Component {
      * @param {string} value - name of the package newly imported
      */
     handleAddImport(value) {
-        const newImportDeclaration = BallerinaASTFactory.createImportDeclaration();
-        newImportDeclaration.setPackageName(value);
-        newImportDeclaration.setParent(this.props.model.parent, { doSilently: true });
-        this.props.model.parent.addImport(newImportDeclaration);
+        if (!value) {
+            return;
+        }
+        value = 'import ' + value + ';\n';
```

Comment:
Just a suggestion. can we add the '\n' as a whitespace region if we have white spaces for this?